app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
-    <background android:drawable="@drawable/ic_launcher_background" />
-    <foreground android:drawable="@drawable/ic_launcher_foreground" />
+<!--    <background android:drawable="@drawable/ic_launcher_background" />-->
```

Comment:
Kunnen deze weg of had je hier het issue mee?

Reply:
Die kunnen weg. Bij een build kreeg ik deze rood te zien dus: missing maar waarschijnlijk zijn deze icoontjes vervangen door Leon. Ze kunnen gewoon weg.

Reply:
Ah oké, kan je deze dan verwijderen en de nieuwe commit pushen? Dan komt de code namelijk in het pull request te staan en kan hij approved worden😉